src/providers/krb5/krb5_child.c

```diff
@@ -933,6 +937,161 @@ static krb5_error_code answer_idp_oauth2(krb5_context kctx,
     return kerr;
 }
 
+static krb5_error_code passkey_preauth(struct krb5_req *kr,
+                                       struct sss_passkey_challenge *passkey)
+{
+    struct krb5_req *tmpkr = NULL;
+    uint32_t offline;
+    errno_t ret;
+
+    if (passkey->domain == NULL || passkey->credential_id_list == NULL ||
```

Comment:
`||` should go in the next line and you should add additional indentation before it.